### PR TITLE
Issue #151: Implement Number-related p0 field actions

### PR DESCRIPTION
**Pull request by jpav**

### CODE REVIEW

**Comment by jpav** [comment]
@igarashitm, Can we change the checkstyle rules so that all instance members exist after static members (including constructors)?

**Comment by igarashitm** [comment]
Sounds good to me, you can add that rule

**Comment by jpav** [comment]
Well, looks like checkstyle only allows me to ignore where constructors show up, which doesn't seem good, so I just moved it to where it wanted it.

File: runtime/core/src/main/java/io/atlasmap/actions/NumberFieldActions.java [hunk 1]
```diff
@@ -238,207 +293,206 @@ public static Number convertVolumeUnit(Action action, Number input) {
...
-            inputs = dinList;
+            return dinList;
         } else if (input instanceof float[]) {
             float[] fin = (float[]) input;
             List<Float> finList = new ArrayList<>(fin.length);
             for (float e : fin) {
                 finList.add(e);
             }
-            if (numberType == null) {
-                numberType = NumberType.FLOAT;
```
Comment (igarashitm) [inline runtime/core/src/main/java/io/atlasmap/actions/NumberFieldActions.java]: Is this field type filling causing any problem?

Comment (igarashitm) [reply runtime/core/src/main/java/io/atlasmap/actions/NumberFieldActions.java]: ah. just noticed it's now in the another method...

**Pull request merged**